jerry-core/ecma/base/ecma-globals.h

```diff
@@ -204,7 +204,7 @@ typedef enum
 
   ECMA_INTERNAL_PROPERTY_NATIVE_HANDLE, /**< native handle associated with an object */
   ECMA_INTERNAL_PROPERTY_FREE_CALLBACK, /**< object's native free callback */
-
+  ECMA_INTERNAL_PROPERTY_NOT_FOUND_AND_STOP, /**< the property is not found and no more search */
```

Comment:
I think we need it, and the it is the same reason that why we need  ECMA_INTERNAL_PROPERTY_NOT_FOUND. because, 'undefined' can be treated the value of a prop
```
var a = {};
a.b = undefined
a.hasOwnProperty('b') // true
Object.getOwnPropertyNames(a) // ['b']
```